ui/app/models/identity/entity.js

```diff
@@ -1,12 +1,21 @@
+import Ember from 'ember';
 import IdentityModel from './_base';
 import DS from 'ember-data';
+import { queryRecord } from 'ember-computed-query';
+const { computed } = Ember;
+
 const { attr, hasMany } = DS;
 
 export default IdentityModel.extend({
-  formFields: ['name', 'policies', 'metadata'],
+  formFields: ['name', 'policies', 'metadata', 'disabled'],
```

Comment:
What do you think about moving disabled to after name? I've been trying to keep the list builders at the bottom so the other fields don't get lost

Reply:
Oh yep, we can do that, the spacing just felt a little strange but I totally forgot about list builders haha. I'll update it!